files/build_templates/init_cfg.json.j2

```diff
@@ -51,7 +51,7 @@
 {%- if include_nat == "y" %}{% do features.append(("nat", "disabled", false, "enabled")) %}{% endif %}
 {%- if include_p4rt == "y" %}{% do features.append(("p4rt", "disabled", false, "enabled")) %}{% endif %}
 {%- if include_restapi == "y" %}{% do features.append(("restapi", "enabled", false, "enabled")) %}{% endif %}
-{%- if include_sflow == "y" %}{% do features.append(("sflow", "disabled", false, "enabled")) %}{% endif %}
+{%- if include_sflow == "y" %}{% do features.append(("sflow", "disabled", true, "enabled")) %}{% endif %}
```

Comment:
Just cross check do we need to modify UT test script for config changes for sflow?
https://github.com/sonic-net/sonic-utilities/blob/1c1e22d3b6123f5635f3ea5f525ed7f1c1d97926/tests/feature_test.py

Reply:
The tests use mock files and hence its not required to modify UT